Formula/s/stringtie.rb

```diff
@@ -0,0 +1,42 @@
+class Stringtie < Formula
+  desc "Transcript assembly and quantification for RNA-Seq"
+  homepage "https://github.com/gpertea/stringtie"
+  url "https://github.com/gpertea/stringtie/archive/refs/tags/v3.0.1.tar.gz"
+  sha256 "aa831451ae08f1ea524db2709d135208695bf66fc2dbcdfb3d1d8461430e2ba9"
+  license "MIT"
+  head "https://github.com/gpertea/stringtie.git", branch: "master"
+
+  depends_on "htslib"
+  depends_on "libdeflate"
+  depends_on "xz"
+
+  uses_from_macos "curl"
+  uses_from_macos "zlib"
```

Comment:
zlib not needed?

https://github.com/Homebrew/homebrew-core/actions/runs/15521631999/job/43695677567#step:5:182

```suggestion
```